guides/v2.1/cloud/cdn/cloud-fastly-custom-response.md

```diff
@@ -0,0 +1,159 @@
+
+The HTML code for the custom page must meet the following requirements:
+
+- Content can contain up to 65,535 characters.
+- Specify all CSS inline in the HTML source.
+- Bundle images in the HTML page using base64 so that they display even if Fastly is offline. See [Data URIs on the css-tricks site](https://css-tricks.com/data-uris/).
+
+
+## Customize the 503 response page
+
+Customers see the default 503 response page in the following cases:
+
+- When a request to the Fastly origin returns a response status great than 500.
+- When the Fastly origin is down, for example due to a timeout, maintenance activity, or health issues.
+
+You can customize the default page by adapting the following HTML code to include styling to match your {{ site.data.var.ee }} store theme and modifying the title and messaging as needed.
+
+```html
+<!DOCTYPE html>
+<html>
+   <head>
+      <meta charset="UTF-8">
+         <title>503</title>
+   </head>
+   <body>
+      <p>Service unavailable</p>
+   </body></html>
+```
+
+Verify that the modified source displays correctly in the browser. Then, add the customized HTML code to the Fastly configuration.
+
+### Add the custom response page to the Fastly configuration
+
+1.  [Log in to the Magento Admin UI]({{ page.baseurl }}/cloud/onboarding/onboarding-tasks.html#admin).
+
+1.  Click **Stores > Settings > Configuration > Advanced > System**.
+
+1.  In the right pane, expand **Full Page Cache > Fastly Configuration > Custom Synthetic Pages**.
+
+    ![Edit 503 error page]
+
+1.	Click **Set HTML**.
+
+1.	Copy and paste the source code for your custom response page into the HTML field. 
+
+    ![Update 503 response]
+	
+1.	Click **Upload** to update the Fastly configuation.
+
+1.	Click **Save Config** at the top of the page.
+
+
+## Customize the WAF response page
+
+If the Magento [Managed Cloud WAF]({{ page.baseurl }}/cloud/cdn/fastly-waf-service.html) blocks a customer request at the Fastly origin, they see the following default response page:
+
+![WAF error response page]
+
+The following code sample shows the HTML source for the default page:
+
+```html
+<html>
+  <head>
+    <title>Magento 403 Forbidden</title>
+  </head>
+  <body>
+    <p>The requested URL was rejected.</p>
+    <p>For additional information, please contact support and provide this reference ID:</p>
+    <p>"} req.http.x-request-id {"</p>
+    <p><button onclick='history.back();'>Go Back</button></p>
+  </body>
+</html>
+```
+
+You can use the **Custom Synthetic Pages > Edit WAF page** option in the Fastly configuration menu to customize the default code for your {{ site.data.var.ece }} project. When you edit the code, retain the following line that provides the reference ID for the WAF blocking event: `<p>"} req.http.x-request-id {"</p>`
+
+
+{:.bs-callout .bs-callout-tip}
+The Edit WAF option is available only if the Managed Cloud WAF service is enabled for your {{ site.var.data.ece }} project.
+
+### Edit the WAF page
```

Comment:
This is another example of procedure title problem. The parent title is "Customize the WAF page.." and this is "Edit the WAF page...". They are very similar, but one is a procedure and one is content. Not so easy to differentiate at a quick glance, so it helps if the procedure is H4 and begins with "To edit..."